### DISCUSSION TITLE
is there a hands-on tutorial for letting bpftrace work on Android Device?

### DISCUSSION THREAD

**Discussion by Katana-O** [bpftrace/bpftrace#1825] [Q&A]
forks, i'm a newcomer of bpftrace/ebpf, just wonder if it is possible to have this tool run on my Android device. Any step-by-step tutorial about setting up the environment for this tool? I'm struggling with picking the correct Device and kernel source version.

**Comment by danobi** [comment]
Unfortunately, there's no tutorials as far as I'm aware. A bit dated, but you might find some hints in https://github.com/iovisor/bpftrace/pull/1095 .
